Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClientContextFixed.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABILITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Build.Tasks;
+using Microsoft.Python.Parsing;
+using Microsoft.PythonTools.Common.Infrastructure;
+using Microsoft.PythonTools.Interpreter;
+using Microsoft.PythonTools.Project;
+using Microsoft.PythonTools.Repl;
+using Microsoft.VisualStudio.InteractiveWindow;
+
+namespace Microsoft.PythonTools.LanguageServerClient {
+    internal class PythonLanguageClientContextFixed : IPythonLanguageClientContext {
+#pragma warning disable CS0067
```

Comment:
does this trick of adding empty add/remove help with the removing the warnings instead of disabling them? https://stackoverflow.com/questions/49994869/suppressing-is-never-used-warning-for-events-in-c-sharp

Reply:
It does, thank you!